### ISSUE TITLE
Holidays in Chile

### ISSUE THREAD

**Issue by ancaneo** [dr-prodigy/python-holidays#328]
Hi.
There are some mistakes in Chile's holidays:
1. Nov 08 is "Asunción de la Virgen"
2. June 29 and October 12' are floating holidays (if it happens on Tuesday, Wednesay or Thursday, the holiday is the preceding Monday. If it happens on Friday, the following Monday is holiday)
3. HolySaturday is holiday ('Semana Santa (Sábado Santo)  [Holy day (Holy Saturday)]')
4. There are 2 localizated holidays, The main administrative division is te region. The ragion codes are:  'AI', 'AN', 'AP', 'AR', 'AT', 'BI', 'CO', 'LI', 'LL', 'LR', 'MA', 'ML', 'NB', 'RM', 'TA', 'VS'.
The localizated holidays are: 
'AP'; {June 7, "Asalto y Toma del Morro de Arica [Assault and Capture of Cape Arica]",}
'NB': {August 20, "Nacimiento de Bernardo O'Higgins [Nativity of Bernardo O'Higgins]"}

Sources:
https://en.wikipedia.org/wiki/ISO_3166-2:CL
https://feriados.cl/
https://feriados.cl/leyes.htm
https://en.wikipedia.org/wiki/Public_holidays_in_Chile

**Comment by mborsetti** [comment]
Thanks for the contribution!  I created pull #373.

1. The correct date for Asunción de la Virgen is 15 AUG (see law 2.977) and was already in the code
2. Confirmed and implemented (law 19.668, http://www.leychile.cl/Navegar?idNorma=160270)
3. Confirmed and implemented (Law 2.977)
4. Arica y Parinacota holiday confirmed with https://feriados.cl/index.php and Law 20.663 (implemented using the correct naming); similarly for Ñuble holiday Law 20.678
